src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_VALUE = 100;
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(MAX_NUMBER_VALUE));
```

Comment:
We don't need create a new instances for classes `Random` and `ColorSupplier` every time when method `getRamdomBall()` is called. Move them to the class fields please and initialize just one time.